code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -0,0 +1,238 @@
+
+//If you kill big slime first oh boy good luck
+/mob/living/simple_animal/hostile/abnormality/melting_love/Life()
+	. = ..()
+	sanityheal()
+	if(!norepeating)
+		if(!bigslime_alive)
+			melee_damage_lower = 62
+			melee_damage_upper = 80
+			adjustBruteLoss(-maxHealth)
+			projectiletype = /obj/projectile/melting_blob/enraged
+			norepeating = TRUE
+		else
+			melee_damage_lower = 32
+			melee_damage_upper = 40
+			projectiletype = /obj/projectile/melting_blob
+	if(bigslime_alive && norepeating)
+		norepeating = FALSE
+
+//Attacks
+/mob/living/simple_animal/hostile/abnormality/melting_love/AttackingTarget()
+	. = ..()
+	if(. && isliving(target))
+		var/mob/living/H = target
+		if(H.stat != DEAD)
+			if(H.health <= HEALTH_THRESHOLD_DEAD && HAS_TRAIT(H, TRAIT_NODEATH))
+				slimeconv(H)
+		else
+			slimeconv(H)
+
+//Slime Conversion
+/mob/living/simple_animal/hostile/proc/slimeconv(mob/living/H)
+	if(!H)
+		return FALSE
+	var/turf/T = get_turf(H)
+	visible_message("<span class='danger'>[src] glomp on \the [H] as another Slime Pawn appears!</span>")
+	H.gib()
+	new /mob/living/simple_animal/hostile/slime(T)
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/melting_love/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 5 SECONDS)
+	QDEL_IN(src, 5 SECONDS)
+	..()
+
+//Qliphoth things
+/mob/living/simple_animal/hostile/abnormality/melting_love/neutral_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/melting_love/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/melting_love/zero_qliphoth(mob/living/carbon/human/user)
+	if(gifted_human)
+		to_chat(gifted_human, "<span class='userdanger'>You feel like you are about to burst !</span>")
+		gifted_human.emote("scream")
+		gifted_human.gib()
+		breach_effect()
+	else
+		breach_effect()
+	return
+
+/* Gift */
+/mob/living/simple_animal/hostile/abnormality/melting_love/proc/GiftedDeath(datum/source, gibbed)
+	SIGNAL_HANDLER
+	var/turf/T = get_turf(gifted_human)
+	gifted_human.gib()
+	new /mob/living/simple_animal/hostile/slime/big(T)
+	datum_reference.qliphoth_change(-9)
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/melting_love/work_complete(mob/living/carbon/human/user, work_type, pe)
+	..()
+	if(work_type != ABNORMALITY_WORK_REPRESSION)
+		if(!gifted_human && istype(user))
+			gifted_human = user
+			RegisterSignal(user, COMSIG_LIVING_DEATH, .proc/GiftedDeath)
+			to_chat(user, "<span class='nicegreen'>You feel like you received a gift...</span>")
+			user.adjust_attribute_buff(TEMPERANCE_ATTRIBUTE, 10)
+			user.add_overlay(mutable_appearance('icons/effects/32x64.dmi', "gift", -HALO_LAYER))
+			playsound(get_turf(user), 'sound/effects/footstep/slime1.ogg', 50, 0, 2)
+			return
+		if(gifted_human)
+			gifted_human.adjustSanityLoss(25)
```

Comment:
Is the idea here to heal sanity whenever work is performed by ANYONE, or to restore sanity of a gifted human when HE is performing it?